src/main/java/digital/slovensko/autogram/core/Autogram.java

```diff
@@ -114,8 +114,7 @@ public void sign(SigningJob job, SigningKey signingKey) {
             } catch (IllegalArgumentException e) {
                 onSigningFailed(AutogramException.createFromIllegalArgumentException(e));
             } catch (ResponseNetworkErrorException e) {
-                ui.onUIThreadDo(() -> ui.closeController(job));
-                onSigningFailed(e);
+                onSigningFailed(e, job);
```

Comment:
Mierne ma metie ten nazov lebo robi nieco ine ako ten onSigningFailed (resp interne ho to zavola). V com sa lisia tie dve volania - skus mi popisat tie scenare.

Reply:
Je tu rozdiel, že pri tomto network errore je potrebné aj zavrieť okno. Nemá zmysel, aby ostalo otvorené. To spojenie už nikdy nevie naspäť ožiť. Otvorené okno by usera mýlilo a skúšal by to zase a zase podpísať bezvýsledne.

onSigningSuccess(job) zatvára okno. onSigningFailed(e) nezatvára, preto som pridal onSigningFailed(e, job), aby zavrel a aby vedel, čo má zavrieť.